gradle.properties

```diff
@@ -8,33 +8,31 @@ yarn_mappings=1.20.1+build.9
 loader_version=0.14.21
 
 # Mod Properties
-mod_version = 0.18.2+1.20
+mod_version = 0.18.3+1.20.1
```

Comment:
Please do not bump the mod version for this project. 
Currently I support Minecraft 1.20 (1.20.0) and 1.20.1 if applicable. I do not know whether you have tested on said version, but if it works then the appended data on the semver is wrong. If, and only if, this version does not work on 1.20, then this would also require a change in the `fabric.mod.json` instead. 
Either way, I have a large list of fixes and plans, which might cause this to become a major patch instead of minor. I therefore prefer to wait with versions bumps until I have planned out a release.